test/integration_test/migration_test.go

```diff
@@ -19,6 +32,11 @@ func testMigration(t *testing.T) {
 	t.Run("failingPreExecRuleTest", migrationFailingPreExecRuleTest)
 	t.Run("failingPostExecRuleTest", migrationFailingPostExecRuleTest)
 	t.Run("labelSelectorTest", migrationLabelSelectorTest)
+	t.Run("migrationIntervalScheduleTest", migrationIntervalScheduleTest)
```

Comment:
Remove the `migration` prefix from the test names. These are subtests under "Migration" so it's redundant.